app/routes/user.js

```diff
@@ -144,6 +146,8 @@ module.exports = [
       }
     },
     handler: async (request, h) => {
+      await sendLoginToAudit(getCallingUser(request), request.headers['enforcement-user-agent'])
```

Comment:
I think this call is used on nearly every GET in enforcement, or at least more regularly than jkust the first login of a session